coremltools/converters/mil/test_flexible_shape_inputs.py

```diff
@@ -23,6 +23,10 @@ def __init__(self, in_channels=3, out_channels=10, kernel_size=3):
         def forward(self, x):
             return self.conv(x)
 
+    class TestFlattenModule(torch.nn.Module):
```

Comment:
Let's add this class definition to the `test_multiarray_input_rangedim_flatten`.